src/Compilers/Core/Portable/PEWriter/MetadataVisitor.cs

```diff
@@ -273,6 +273,7 @@ public virtual void Visit(IMethodDefinition method)
         {
             this.Visit(method.GetReturnValueAttributes(Context));
             this.Visit(method.ReturnValueCustomModifiers);
+            this.Visit(method.RefCustomModifiers);
```

Comment:
It looks like there is one more place in ReferenceDependencyWalker.cs as well.

---
In reply to: [118057254](https://github.com/dotnet/roslyn/pull/19658#discussion_r118057254) [](ancestors = 118057254)